docs/topics/auth/passwords.txt

```diff
@@ -62,10 +62,11 @@ The default for :setting:`PASSWORD_HASHERS` is::
         'django.contrib.auth.hashers.PBKDF2SHA1PasswordHasher',
         'django.contrib.auth.hashers.Argon2PasswordHasher',
         'django.contrib.auth.hashers.BCryptSHA256PasswordHasher',
+	'django.contrib.auth.hashers.ScryptPasswordHasher',
```

Comment:
This isn't accurate based on your latest edit (removal from `global_settings.py`).

Reply:
Removed `ScryptPasswordHasher ` lines from passwords.txt documentation.